frontend/src/components/common/Header/Profile/Mobile.svelte

```diff
@@ -1,7 +1,6 @@
 <script lang="ts">
   import Icon from '$components/icons/Icon.svelte';
   import type { UserMenuItem } from './Desktop.svelte';
-
```

Comment:
What do you think about module and logical separation between block or module?

Reply:
I dont understand what you do want...
I prefer discuss that in discord

> What do you think about module and logical separation between block or module?